app/serializers/project_serializer.rb

```diff
@@ -1,8 +1,16 @@
 # frozen_string_literal: true
 class ProjectSerializer < ActiveModel::Serializer
-  attributes :id, :name, :url, :permalink, :repository_url
+  attributes :id, :name, :url, :permalink, :repository_url, :owner, :created_at
+
+  def self.csv_header
```

Comment:
this relies on the ordering of attributes and csv_line to be always the same, which is not tested ... how about 
```
PROPERTIES = [:id, :foo]
attributes *PROPERTIES

def csv_header
  PROPERTIES.map { |p| p.to_s.humanize }
end

def csv_line
  PROPERTIES.each { |p| public_send p }
end
```